File: QuisineAngular/src/app/models/customer.ts
export class Customer {
    constructor(
      public userId: number = 0,
      public userName: string = '',
      public userEmail: string = '',
      public userPass: string = '',
      public userImg: string = '',
      public userContact: string = '',
      public userAddress: string = '',
      public userCity: string = '',
      public userState: string = '',
      public userPin: number = 0,
      public role: string = '',
      public isBlocked:boolean=false

      ) {}
}

File: QuisineSpringBoot/Quisine/src/main/java/com/project/Quisine/dto/Customer.java
package com.project.Quisine.dto;

import lombok.Data;

@Data
public class Customer {

    private int userId;
	private String userName;
	private String userEmail;
	private String userPass;
	private String userImg;
	private String userContact;
	private String userAddress;
	private String userCity;
	private String userState;
	private int userPin;
	private String role;
}
